src/ui_bagl.c

```diff
@@ -106,6 +136,10 @@ static bool calculate_baking_idle_screens_data(void) {
 
     TZ_ASSERT(hwm_to_string(home_context.hwm, sizeof(home_context.hwm), &N_data.hwm.main) >= 0,
               EXC_WRONG_LENGTH);
+    TZ_ASSERT(hwm_status_to_string(home_context.hwm_status,
+                                   sizeof(home_context.hwm_status),
+                                   &N_data.hwm_disabled) >= 0,
+              EXC_WRONG_LENGTH);
```

Comment:
Why do we need to update the `hwm_status` here?

Reply:
We dont. removed it. Also rebased on main.